pipelines/release-to-github-signed/release-to-github-signed.yaml

```diff
@@ -0,0 +1,348 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: release-to-github-signed
+  labels:
+    app.kubernetes.io/version: "1.0.0"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton release pipeline to release binaries extracted from the image built with RHTAP, signed a checksum file, and
+    release the binaries and signed file to GitHub.
+  params:
```

Comment:
Yesterday, we merged this PR that added git resolver parameters to each pipeline: https://github.com/redhat-appstudio/release-service-catalog/pull/298

So these parameters should be added here (along with using them in the task references).

Reply:
Updated!